tools/devtool

```diff
@@ -1068,12 +1068,15 @@ cmd_shell() {
 
     # By default, we run the container as the current user.
     privileged=false
+    cmd="echo \"Specify a command\""
+    execute_cmd=false
 
     # Parse any command line args.
     while [ $# -gt 0 ]; do
         case "$1" in
             "-h"|"--help")          { cmd_help; exit 1; } ;;
             "-p"|"--privileged")    { privileged=true;  } ;;
+            "-c"|"--command")       { cmd=$2; execute_cmd=true;shift;shift;shift;break;  } ;;
```

Comment:
Why is the triple `shift` needed?

Reply:
Right, we only need two of them. Updated.